build.sbt

```diff
@@ -1,4 +1,6 @@
-lazy val scalaz = ProjectRef(uri("git:https://github.com/scalaz/scalaz.git#series/8.0.x"), "base")
+resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
```

Comment:
Can you explain what this resolver is needed for?

Reply:
It’s needed for `zio` which is a dependency of `baseJVM`. I don’t know if there is a better way to resolve it to be honest

Reply:
Alright, that make perfect sense (I didn't think about that before).
I've made [a PR](https://github.com/scalaz/scalaz-sbt/pull/18) to the plugin so that projects don't need to explicitly add this resolver. 
I think we should remove this setting after the said PR is merged.

Reply:
The said PR has been merged on scalaz-sbt

Reply:
Nice one - I'll remove it from here then